src/controller/CommissionerDiscoveryController.cpp

```diff
@@ -244,6 +239,18 @@ void CommissionerDiscoveryController::InternalOk()
             mUserPrompter->PromptForAppInstallOKPermission(client->GetVendorId(), client->GetProductId(), client->GetDeviceName());
         }
         ChipLogDetail(Controller, "------Via Shell Enter: app install <pid> <vid>");
+
+        CommissionerDeclaration::CdError appInstallStatus =
+            mAppInstallationService->GetInstallationStatusOfApp(client->GetVendorId(), client->GetProductId());
+
+        if (appInstallStatus != CommissionerDeclaration::CdError::kNoError)
```

Comment:
We need to return something to the casting client, even when there is no entry in the install status table. At the very least it should be the kAppInstallConsentPending

Reply:
Removed the if statement.